CHANGELOG/CHANGELOG-2.3.md

```diff
@@ -0,0 +1,58 @@
+- Support providing a fixed public host key for SFTP uploads with a new field `hostPublicKey` to `PacketCapture` and `SupportBundleCollection` CRDs. ([#6848](https://github.com/antrea-io/antrea/pull/6848), [@antoninbas])
+- Skip processing Service without IPPool annotation `service.antrea.io/external-ip-pool` for `ServiceExternalIP` feature. ([#6950](https://github.com/antrea-io/antrea/pull/6950), [@xliuxu])
+- Improve the document of the configuration option `egress.exceptCIDRs`. ([#6828](https://github.com/antrea-io/antrea/pull/6828), [@tnqn])
+- Upgrade CNI plugins from v1.5.1 to v1.6.2. ([#6796](https://github.com/antrea-io/antrea/pull/6796), [@luolanzone])
+- Push Antrea Ubuntu-based images to `ghcr.io`. ([#6834](https://github.com/antrea-io/antrea/pull/6834), [@antoninbas])
+- Upgrade go-ipfix to 0.13.0. ([#6998](https://github.com/antrea-io/antrea/pull/6998), [@antoninbas])
+
+### Fixed
+
+- Add `-ComputerName localhost` explicitly for VMSwitch commands to avoid potential validation issue on Windows with Active Directory. ([#6985](https://github.com/antrea-io/antrea/pull/6985), [@XinShuYang])
+- Fix that Antrea L7NetworkPolicies do not handle Service traffic correctly. ([#6941](https://github.com/antrea-io/antrea/pull/6941), [@hongliangl])
+- Disable TX checksum offload for Antrea host gateway interface when `disableTXChecksumOffload` is set to `true`. ([#6843](https://github.com/antrea-io/antrea/pull/6843), [@hongliangl])
+- Add `fqdnCacheMinTTL` configuration for Antrea ClusterNetworkPolicy to ensure that resolved IPs are included in data path rules as long as the application caches them. ([#6808](https://github.com/antrea-io/antrea/pull/6808), [@hkiiita])
+- Support OF port state "down" when installing forwarding rules for Pods on Windows. ([#6889](https://github.com/antrea-io/antrea/pull/6889), [@wenyingd])
+- Fix audit logging for default deny all K8s NetworkPolicy rules. ([#6855](https://github.com/antrea-io/antrea/pull/6855), [@qiyueyao])
+- Ensure that `promote_secondaries` is set on `IPAssigner` interfaces to avoid the automatic removal of all other IP addresses in the same subnet when the primary IP address is deleted. ([#6898](https://github.com/antrea-io/antrea/pull/6898) [#6900](https://github.com/antrea-io/antrea/pull/6900), [@antoninbas])
+- Set the maximum packet size explicitly to fix an issue with reading `PacketCapture` pcapng files in `tcpdump` on macOS. ([#6804](https://github.com/antrea-io/antrea/pull/6804), [@hangyan])
+- Reconcile Pods with `hostNetwork` after Antrea Agent is restarted on Windows. ([#6944](https://github.com/antrea-io/antrea/pull/6944), [@wenyingd])
+- Create a new kubeconfig for `SupportBundleClient` to fix `antctl supportbundle` failures on Windows. ([#6840](https://github.com/antrea-io/antrea/pull/6840), [@XinShuYang])
+- Fix race condition when getting BGP routes in BGPController. ([#6823](https://github.com/antrea-io/antrea/pull/6823), [@Atish-iaf])
```

Comment:
maybe we could omit this since the functionality is new in this release

alternatively, you can add this PR number to the "Add route info in the output of `antctl get bgproutes`" entry

Reply:
Move it with the "Add route info in the output of antctl get bgproutes" entry.